scripts/test_S1_performance.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+function join_by {
+  local d=${1-} f=${2-}
+  if shift 2; then
+    printf %s "$f" "${@/#/$d}"
+  fi
+}
+# Initialise EESSI
+source /cvmfs/pilot.eessi-hpc.org/latest/init/bash > /dev/null
+# Grab the date we do this and use that as out key for json output
+date=$(date -I)
+echo "{ \"$date\":["
+json_array=()
+for s1server in `grep CVMFS_SERVER_URL /etc/cvmfs/domain.d/eessi-hpc.org.conf | grep -o '".*"' | sed 's/"//g' | tr ';' '\n'`; do
+    # Edit the config file to point to a single S1 option, e.g.,
+    echo 'CVMFS_SERVER_URL="'"$s1server"'"' | sudo tee /etc/cvmfs/domain.d/eessi-hpc.org.local > /dev/null
+    # Reconfigure CVMFS 
+    sudo cvmfs_config setup
```

Comment:
Should have a way to check the user has the required `sudo` rights

Reply:
Wasn't the idea to run this in a container, with bind mounts to empty directories for `/var/lib/cvmfs` & co (so empty cache), which would totally alleviate the need for `sudo`?

Reply:
It wasn't my idea...

Reply:
It actually doesn't seem like running it in a container can work:
```bash
Apptainer> cvmfs_config stat pilot.eessi-hpc.org    
pilot.eessi-hpc.org not mounted
Apptainer> source /cvmfs/pilot.eessi-hpc.org/latest/init/bash 
Found EESSI pilot repo @ /cvmfs/pilot.eessi-hpc.org/versions/2021.12!
archspec says aarch64/graviton2
Using aarch64/graviton2 as software subdirectory.
Using /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/aarch64/graviton2/modules/all as the directory to be added to MODULEPATH.
Found Lmod configuration file at /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/aarch64/graviton2/.lmod/lmodrc.lua
Initializing Lmod...
Prepending /cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/aarch64/graviton2/modules/all to $MODULEPATH...
Environment set up to use EESSI pilot software stack, have fun!
[EESSI pilot 2021.12] $ cvmfs_config stat pilot.eessi-hpc.org
pilot.eessi-hpc.org not mounted
[EESSI pilot 2021.12] $ cvmfs_config wipecache
root privileges required
[EESSI pilot 2021.12] $ cvmfs_config setup    
root privileges required
```
I can get away with not wiping the cache since I control it, and I guess I can just overwrite the EESSI configuration file with a bind mount, but I am only left with timing the command if I can't run `cvmfs_config stat`

Reply:
Actually, more than that, I need to be able to run `cvmfs_config setup` to put the new configuration in place. Otherwise I need to bind mount a new configuration and run each S1 in a separate container execution step.

I really don't see the worth, we lose a lot of valuable info and this implentation will run just fine as a CI job.